src/Command/Proxy/RunSqlCommand.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Doctrine\Bundle\DBALBundle\Command\Proxy;
+
+use Doctrine\DBAL\Tools\Console\Command\RunSqlCommand as DoctrineRunSqlCommand;
+use Doctrine\DBAL\Tools\Console\Helper\ConnectionHelper;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Execute a SQL query and output the results.
+ */
+class RunSqlCommand extends DoctrineRunSqlCommand
```

Comment:
actually since https://github.com/doctrine/dbal/pull/3956 no need to extend the command anymore.

So we should require dbal 2.11 and inject an adapter from our registry to the `ConnectionProvider` interface

Reply:
For now I changed it so we support both < 2.11 and >= 2.11 without deprecations.

Or should we just require >= 2.11 already?

Reply:
Depends on the ETA for DBAL 2.11. If it is released soon enough for us to require it, I would go for that option.

Symfony 5.1 will drop support for DBAL <2.10 anyway and older DBAL version are not really maintained, so I'm fine with asking people to use an uptodate DBAL version when switching to our new bundle.